src/blueprints/tv2/action-factories/tv2-graphics-action-factory.ts

```diff
@@ -150,4 +199,322 @@ export class Tv2GraphicsActionFactory {
+    let chosenMethod: (blueprintConfiguration: Tv2BlueprintConfiguration, graphicsData: Tv2GraphicsData) => Tv2PartAction
+    switch (blueprintConfiguration.studio.GraphicsType) {
+      case Tv2GraphicsType.HTML: {
+        chosenMethod = this.createCasparCgFullGraphicsActionFromGraphicsData.bind(this)
+        break
+      }
+      case Tv2GraphicsType.VIZ:
+      default: {
+        chosenMethod = this.createVizFullGraphicsActionFromGraphicsData.bind(this)
+        break
+      }
+    }
+
+    return graphicsData.map((data) => chosenMethod(
+      blueprintConfiguration,
+      data
+    ))
+  }
+
+  private createVizFullGraphicsActionFromGraphicsData(blueprintConfiguration: Tv2BlueprintConfiguration, graphicsData: Tv2GraphicsData): Tv2PartAction {
+    const partId: string = 'fullGraphicsPart'
+    const fullGraphicPiece: PieceInterface = this.createVizFullGraphicsPiece(blueprintConfiguration, graphicsData, partId)
+    return this.createFullGraphicsActionFromGraphicsData(
+      blueprintConfiguration.studio.VizPilotGraphics.KeepAliveDuration,
+      partId,
+      fullGraphicPiece,
+      graphicsData
+    )
+  }
+
+  private createFullGraphicsActionFromGraphicsData(
+    keepPreviousPartAliveDuration: number,
+    partId: string,
+    pieceInterface: PieceInterface,
+    graphicsData: Tv2GraphicsData
+  ): Tv2PartAction {
+    const partInterface: PartInterface = this.createGraphicsPartInterface({
+      id: partId,
+      name: `Full ${graphicsData.name}`,
+      inTransition: {
+        keepPreviousPartAliveDuration,
+        delayPiecesDuration: 0
+      },
+    })
+
+    return {
+      id: `full_graphics_${this.getHashedValue(graphicsData.name)}`,
+      name: `Full Graphics - ${graphicsData.name}`,
+      type: PartActionType.INSERT_PART_AS_NEXT,
+      data: {
+        partInterface: partInterface,
+        pieceInterfaces: [pieceInterface]
+      },
+      metadata: {
+        contentType: Tv2ActionContentType.GRAPHICS
+      }
+    }
+  }
+
+  // Todo: Figure out if it should be moved to a base class for all action factories
+  private getHashedValue(valueToBeHashed: string ): string {
+    return createHash('md5').update(valueToBeHashed).digest('hex')
+  }
+
+  private createVizFullGraphicsPiece(blueprintConfiguration: Tv2BlueprintConfiguration, graphicsData: Tv2GraphicsData, partId: string): PieceInterface {
+    return this.createGraphicsPieceInterface({
+      id: 'fullGraphicPiece',
+      partId,
+      name: graphicsData.name,
+      preRollDuration: blueprintConfiguration.studio.VizPilotGraphics.PrerollDuration,
+      pieceLifespan: this.findInfiniteModeFromConfig(blueprintConfiguration, graphicsData),
+      layer: Tv2SourceLayer.PILOT_GRAPHICS,
+      content: {
+        fileName: `PILOT_${graphicsData.vcpId}`,
+        path: `${graphicsData.vcpId}`,
+      },
+      timelineObjects: [
+        this.vizGraphicsTimelineObjectFactory.createFullGraphicsTimelineObject(blueprintConfiguration, graphicsData),
```

Comment:
While moving over from OG Blueprints, i was having a hard time seeing if i had moved the correct timeline objects over. 
Because of this, i chose to setup how the Timeline Objects were created in the OG Blueprints in the new setup.

Reply:
Creation of that one timeline object has been moved into the `createVizFullPilotTimelineObjects` method.